Source/TrainProject/Classes/Passenger.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System.Collections.Generic;
+using System.IO;
+
+namespace TrainProject
+{
+    public class Passenger
+    {
+        public int id { get; }
+        public string name { get; }
+
+        public Passenger(string passengerList)
+        {
+            string[] convert = passengerList.Split(";");
+            id = int.Parse(convert[0]);
+            name = convert[1];
+        }
+    }
+
+    public class PassengerList
+    {
+        public List<Passenger> InitAllPassengers()
+        {
+            List<Passenger> allPassengers = new List<Passenger>();
+            string[] passengers = File.ReadAllLines(Program.PassengersFilePath);
+
+            foreach (string item in passengers)
+            {
+                allPassengers.Add(new Passenger(item));
+            }
+
+            return allPassengers;
+        }
+    }
```

Comment:
I think that an entire class for one method is too much. Same with "InitAvailableTrain()"-method and the other ones. I don't think it's necessary to create a class and an instance just to load passenger data. You might want to create a class which has the responsibility to load data from files and static methods to retrieve data.

Reply:
If it the method does not fit in anywhere else, and the program in general is fulfilling single responsibility, doe it not matter in my eyes how many classes one creates, the overhead is extremely small.
I wonder what InitAllPassengers does? If I read the code it reads a file and adds a new passenger object foreach row. A better name would maybe be: ReadPassengersFromFile.
Then the question is the name of the class... or if this method belongs in another class.